website/server/controllers/api-v3/challenges.js

```diff
@@ -21,18 +21,153 @@ import csvStringify from '../../libs/csvStringify';
 let api = {};
 
 /**
+ * @apiDefine ChallengeLeader Challenge Leader
+ * The leader of the challenge can use this route.
+ */
+
+/**
  * @apiDefine ChallengeNotFound
  * @apiError (404) {NotFound} ChallengeNotFound The specified challenge could not be found.
  */
 
 /**
+ * @apiDefine SuccessfulChallengeRequest
+ * @apiSuccess {UUID} challenge.group._id The group id.
+ * @apiSuccess {String} challenge.group.type Group type: `Guild` or `Party`.
+ * @apiSuccess {String} challenge.group.privacy Group privacy: `Public` or `Private`.
+ * @apiSuccess {String} challenge.name Full name of challenge.
+ * @apiSuccess {String} challenge.shortName Short name of challenge.
+ * @apiSuccess {Object} challenge.leader User details of challenge leader.
+ * @apiSuccess {UUID} challenge.leader._id User id of challenge leader.
+ * @apiSuccess {Object} challenge.leader.profile Profile information of leader.
+ * @apiSuccess {Object} challenge.leader.profile.name Username of leader.
+ * @apiSuccess {String} challenge.updatedAt Timestamp of last update.
+ * @apiSuccess {String} challenge.createdAt Timestamp of challenge creation.
+ * @apiSuccess {UUID} challenge.id Id number of newly created challenge.
+ * @apiSuccess {UUID} challenge._id Same as `challenge.id`.
+ * @apiSuccess {String} challenge.prize Number of gems offered as prize to winner.
+ * @apiSuccess {String} challenge.memberCount Number users participating in challenge.
+ * @apiSuccess {Object} challenge.tasksOrder Object representing associated tasks and rewards.
```

Comment:
`challenge.tasksOrder` would be better described as "Object containing IDs of the challenge's tasks and rewards in their preferred sort order"
(The sorting doesn't currently work but we'll fix that.)

For `challenge.tasksOrder.rewards`, use "Array of \`reward\` task IDs", and make similar changes for the other three task types.